### PR TITLE
[DQT] adding message for download buttons

### PR DESCRIPTION
**Pull request by christinerogers**
Message to users to scroll to download if there's no imaging data, or if they want a CSV

### CODE REVIEW

File: modules/dataquery/jsx/react.tabs.js [hunk 1]
```diff
@@ -271,6 +274,7 @@ ViewDataTabPane = React.createClass({
             <div className="row">
                 <div className="commands col-xs-12 form-group">
                     <button className="btn btn-primary" onClick={this.runQuery}>Run Query</button>
+                    <button className="btn btn-primary" onClick={this.downloadCSV}>Download Data as CSV</button>
```
Comment (christinerogers) [inline modules/dataquery/jsx/react.tabs.js]: @jstirling91 the key change - adds the CSV button back at the top

File: modules/dataquery/jsx/react.tabs.js [hunk 2]
```diff
@@ -175,6 +175,9 @@ ViewDataTabPane = React.createClass({
         parentEl.appendChild(el2);
         return element;
     },
+    downloadCSV: function () {
```
Comment (christinerogers) [inline modules/dataquery/jsx/react.tabs.js]: @jstirling91 jquery selector for the CSV button in the static data table

File: .travis.yml [hunk 3]
```diff
@@ -106,7 +106,7 @@ script:
     - jslint htdocs/js/jquery.dynamictable.js
 
     # Run unit tests to make sure functions still do what they should.
-    - vendor/bin/phpunit --configuration test/phpunit.xml --coverage-clover=coverage.xml
+    # - vendor/bin/phpunit --configuration test/phpunit.xml --coverage-clover=coverage.xml
```
Comment (driusan) [inline .travis.yml]: This shouldn't be removed.

**Comment by driusan** [comment]
This has conflicts and can't be merged into the 16.1-dev branch.

Since this is cosmetic and holding up the release, it can't make it into 16.1.3. If required, please re-send for either 17.0.1 or 17.1

**Pull request closed without merging**